platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,30 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    package_managers="apt-get dpkg yum rpm"
+    for package in $package_managers; do
+    {
+        res="$(need_cmd $package)"  
+        if [ $? -eq 0 ] ; then
+            not_found=0
+            wrap_debug "$package package manager is present."
```

Comment:
we should also record what package manager/ package Management system is supported

Reply:
where should we record it? When no one of the package manager is found, then we can record it?